openeo/testing/results.py

```diff
@@ -230,9 +233,17 @@ def _compare_xarray_dataarray(
         issues.append(f"Shape mismatch: {actual.shape} != {expected.shape}")
     compatible = len(issues) == 0
     try:
-        xarray.testing.assert_allclose(a=actual, b=expected, rtol=rtol, atol=atol)
+        if (pixel_tolerance > _DEFAULT_PIXELTOL) and compatible:
```

Comment:
why comparing with `_DEFAULT_PIXELTOL`? It's just about being nonzero, right?
```suggestion
        if pixel_tolerance and compatible:
```

Reply:
ahh yes.